src/yayson/presenter.coffee

```diff
@@ -81,7 +85,7 @@ module.exports = (utils, adapter) ->
         collection = instanceOrCollection
         @scope.data ||= []
         collection.forEach (instance) =>
-          @toJSON instance
+          @toJSON instance, options
```

Comment:
The test does not pass when removing the options from that line, because when the relation is an array the include option should be considered for each of the members.